### PR TITLE
Template update

### PR DESCRIPTION
**Pull request by luisas**
## PR checklist

Template update PR

### CODE REVIEW

File: modules.json [hunk 1]
```diff
@@ -36,6 +35,11 @@
                         "git_sha": "666652151335353eef2fcd58880bcef5bc2928e1",
                         "installed_by": ["modules"]
                     },
+                    "fastqc": {
+                        "branch": "master",
+                        "git_sha": "666652151335353eef2fcd58880bcef5bc2928e1",
+                        "installed_by": ["modules"]
+                    },
```
Comment (mirpedrol) [inline modules.json:42]: you don't want fastqc in the pipeline, right?
```suggestion
```

File: modules/nf-core/fastqc/environment.yml
Comment (mirpedrol) [inline modules/nf-core/fastqc/environment.yml:1]: remove the fastqc module

File: modules.json [hunk 2]
```diff
@@ -23,8 +23,7 @@
                     "csvtk/join": {
                         "branch": "master",
                         "git_sha": "666652151335353eef2fcd58880bcef5bc2928e1",
-                        "installed_by": ["modules"],
-                        "patch": "modules/nf-core/csvtk/join/csvtk-join.diff"
```
Comment (mirpedrol) [inline modules.json:27]: I'm not sure if removing all patch files was intended or an artifact of the template update, leaving this comment to make sure that this is not a mistake. It also happens with other modules 👀

File: modules/nf-core/tcoffee/align/main.nf [hunk 3]
```diff
@@ -39,14 +39,6 @@ process TCOFFEE_ALIGN {
...
         $write_output
 
-    # If stdout file exist and compress is true, then compress the file
-    # This is a patch for the current behaviour of the regressive algorithm
-    # that does not support the stdout redirection
-    if [ -f stdout ] && [ "$compress" = true ]; then
-        pigz -cp ${task.cpus} < stdout > ${prefix}.aln.gz
-        rm stdout
-    fi
-
```
Comment (mirpedrol) [inline modules/nf-core/tcoffee/align/main.nf:49]: was it intended to remove this?

File: nextflow.config [hunk 4]
```diff
@@ -203,19 +185,28 @@ profiles {
...
+// Load nf-core custom profiles from different Institutions
+includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/nfcore_custom.config" : "/dev/null"
 
 // Nextflow plugins
 plugins {
     id 'nf-validation@1.1.4' // Validation of pipeline parameters and creation of an input channel from a sample sheet
 }
 
+// Load nf-core/multiplesequencealign custom profiles from different institutions.
+// includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/pipeline/multiplesequencealign.config" : "/dev/null"
```
Comment (mirpedrol) [inline nextflow.config:197]: You can uncomment this line, it's to allow the use of an nf-core/config specific for this piepeline

File: subworkflows/local/utils_nfcore_multiplesequencealign_pipeline/main.nf [hunk 5]
```diff
@@ -77,8 +67,8 @@ workflow PIPELINE_INITIALISATION {
     //
     // Create channel from input file provided through params.input
     //
-    ch_input = Channel.fromSamplesheet('input')
-    ch_tools = Channel.fromSamplesheet('tools')
+    ch_input = Channel.fromList(samplesheetToList(params.input, "${projectDir}/assets/schema_input.json"))
+    ch_tools = Channel .fromList(samplesheetToList(params.tools, "${projectDir}/assets/schema_tools.json"))
```
Comment (mirpedrol) [inline subworkflows/local/utils_nfcore_multiplesequencealign_pipeline/main.nf:71]:
```suggestion
    ch_tools = Channel.fromList(samplesheetToList(params.tools, "${projectDir}/assets/schema_tools.json"))
```

File: workflows/multiplesequencealign.nf [hunk 6]
```diff
@@ -197,45 +194,58 @@ workflow MULTIPLESEQUENCEALIGN {
...
+    summary_params      = paramsSummaryMap(
+        workflow, parameters_schema: "nextflow_schema.json")
+    ch_workflow_summary = Channel.value(paramsSummaryMultiqc(summary_params))
+    ch_multiqc_files = ch_multiqc_files.mix(
+        ch_workflow_summary.collectFile(name: 'workflow_summary_mqc.yaml'))
+    ch_multiqc_custom_methods_description = params.multiqc_methods_description ?
+        file(params.multiqc_methods_description, checkIfExists: true) :
+        file("$projectDir/assets/methods_description_template.yml", checkIfExists: true)
+    ch_methods_description                = Channel.value(
+        methodsDescriptionText(ch_multiqc_custom_methods_description))
```
Comment (mirpedrol) [inline workflows/multiplesequencealign.nf:224]: the equal signs could be aligned 🙂

File: workflows/multiplesequencealign.nf [hunk 7]
```diff
@@ -197,45 +194,58 @@ workflow MULTIPLESEQUENCEALIGN {
...
+    MULTIQC (
+        ch_multiqc_files.collect(),
+        ch_multiqc_config.toList(),
+        ch_multiqc_custom_config.toList(),
+        ch_multiqc_logo.toList(),
+        [],
+        []
+    )
+
+    emit:multiqc_report = MULTIQC.out.report.toList() // channel: /path/to/multiqc_report.html
```
Comment (mirpedrol) [inline workflows/multiplesequencealign.nf:246]:
```suggestion
    emit:
    multiqc_report = MULTIQC.out.report.toList() // channel: /path/to/multiqc_report.html
```

**Review by mirpedrol — APPROVED** [review]
I made some style comments. 
But mostly, I am not sure if the template update removed all your patch files from modules, so please make sure that everything is as it should be 🙂 
After this is addressed, LGTM 🚀

**Comment by luisas** [comment]
> I made some style comments. But mostly, I am not sure if the template update removed all your patch files from modules, so please make sure that everything is as it should be 🙂 After this is addressed, LGTM 🚀

Thanks a lot for the review!!

File: .gitignore [hunk 8]
```diff
@@ -7,8 +7,3 @@ testing/
 testing*
 *.pyc
 null/
-outdir/
-.ipynb_checkpoints/
-test.ipynb
-*reports.tsv
-.nf-test*
```
[the remark this answers is not in the document]
Comment (luisas) [reply .gitignore:14]: Yes they were in my pipeline but actually should not have been, they were specific to my runs locally :D

File: nextflow.config (hunk 4 above)
Comment (luisas) [reply nextflow.config:197]: ok

File: modules.json (hunk 2 above)
Comment (luisas) [reply modules.json:27]: UHHH massive catch!! Thanks :D

File: modules/nf-core/tcoffee/align/main.nf (hunk 3 above)
Comment (luisas) [reply modules/nf-core/tcoffee/align/main.nf:49]: yes :) i moved the regressive (the one needing this lines) to a new module

**Pull request merged**